sdk/tests/types/output_id.rs

```diff
@@ -3,19 +3,30 @@
 
 use core::str::FromStr;
 
-use iota_sdk::types::block::{output::OutputId, payload::transaction::TransactionId, Error};
+use iota_sdk::types::block::{
+    output::OutputId,
+    payload::transaction::{TransactionHash, TransactionId},
+    Error,
+};
 use packable::{bounded::InvalidBoundedU16, error::UnpackError, PackableExt};
 use pretty_assertions::assert_eq;
 
-const TRANSACTION_ID: &str = "0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c649";
-const OUTPUT_ID: &str = "0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c6492a00";
-const OUTPUT_ID_INVALID_INDEX: &str = "0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c6498000";
+const TRANSACTION_ID: &str = "0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c64900000000";
+const OUTPUT_ID: &str = "0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c649000000002a00";
+const OUTPUT_ID_INVALID_INDEX: &str = "0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c649000000008000";
 
 #[test]
 fn debug_impl() {
     assert_eq!(
         format!("{:?}", OutputId::from_str(OUTPUT_ID).unwrap()),
-        "OutputId(0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c6492a00)"
+        "OutputId { \
+            id: \"0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c649000000002a00\", \
+            transaction_id: TransactionId { \
+                id: \"0x52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c64900000000\", \
```

Comment:
Looks weird, this already has the slot index encoded in it, no?

Reply:
Oh boy, here we go again

Reply:
Ok I see you wanted it to easily copy it, fine with that
Just looked weird together with the slot_index, because then one could think the slot:index gets added as well for the full id

Reply:
There, I said it again :trollface: